kb-dk/ds-web
# Disc 506/package maintenance 20231128

Finally got this working. It took a loooooot of tinkering.

Got rid of the need to run `npm i `with` --legacy-peer-deps` or `--force` \o/ and had to make som slight code changes and changes to configs.

To test:

1. Check the branch out and clear you current /node_modules
2. Run npm i
3. Try to run npm run serve and check if the app is still good
4. Finally try to build the project with npm run build
